app/views/content/start-teacher-training-this-september.md

```diff
@@ -7,6 +7,16 @@ description: |-
 date: "2021-06-17"
 image: "media/images/content/hero-images/0008.jpg"
 backlink: "../"
+right_column:
+  ctas:
+    - title: Sign up for our event
+      text: |
+       Get tips from our experts on making a strong application.  
+      link_text: "sign up"
+      link_target: "/start-teacher-training-this-september"
```

Comment:
@Tomcowan1 do we have the event link to push to?

Reply:
Hi @MylesJarvis not yet - was just trying to get a PR on the go to see what it would look like, and share with the on-campus team (and see if I could figure out how to add a side box on the site). It doesn't need to go live until next Tues.